api/instructor/instructorRouter.js

```diff
@@ -2,11 +2,49 @@ const express = require('express');
 const authRequired = require('../middleware/authRequired');
 const Instructors = require('./instructorModel');
 const router = express.Router();
+const Profiles = require('../profile/profileModel');
+const oktaClient = require('../../lib/oktaClient');
 const {
   getInstructorId,
   checkInstructorExist,
 } = require('./instructorMiddleware');
 
+/* Create a new Instructor profile */
+router.post('/register', (req, res) => {
+  if (!req.body) return res.sendStatus(400);
+  const newInstructor = {
+    profile: {
+      firstName: req.body.firstName,
+      lastName: req.body.lastName,
+      email: req.body.email,
+      login: req.body.email,
+    },
+    credentials: {
+      password: {
+        value: req.body.password,
```

Comment:
We also don't want to be attempting to create a user with a password, I don't think! Okta should allow us to create a user without a given password, emailing them to have them complete their registration and pick their password then.

Reply:
Added these changes! Also, cleaned up console.logs